### PR TITLE
git-interactive-rebase-tool: new, 2.4.1

### PR DESCRIPTION
**Pull request by xtexx**
Topic Description
-----------------

- git-interactive-rebase-tool: lint scripts
- git-interactive-rebase-tool: bump REL for topic Revision Marking Guidelines
- git-interactive-rebase-tool: new, 2.4.1

Package(s) Affected
-------------------

- git-interactive-rebase-tool: 2.4.1-1

Security Update?
----------------

No

Build Order
-----------

```
#buildit git-interactive-rebase-tool
```

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`
- [x] AArch64 `arm64`
- [x] LoongArch 64-bit `loongarch64`

**Secondary Architectures**

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`
- [x] RISC-V 64-bit `riscv64`

### CODE REVIEW

File: app-devel/git-interactive-rebase-tool/autobuild/defines [hunk 1]
```diff
@@ -0,0 +1,8 @@
+PKGNAME=git-interactive-rebase-tool
+PKGDES="An improved sequence editor for Git"
+PKGSEC="devel"
+PKGDEP="glibc"
+BUILDDEP="rustc llvm"
+
+CARGO_AFTER="--features=git2/vendored-libgit2"
+
```
Comment (eatradish) [inline app-devel/git-interactive-rebase-tool/autobuild/defines:6]: Useless empty last line

File: app-devel/git-interactive-rebase-tool/spec [hunk 2]
```diff
@@ -0,0 +1,5 @@
+VER=2.4.1
+SRCS="git::commit=tags/$VER::https://github.com/MitMaro/git-interactive-rebase-tool.git"
+CHKSUMS="SKIP"
+CHKUPDATE="anitya::id=181957"
+
```
Comment (eatradish) [inline app-devel/git-interactive-rebase-tool/spec]: Useless empty last line

**Review by eatradish — DISMISSED** [review]

**Comment by xtexx** [comment]
lgtm on amd64
image

File: app-devel/git-interactive-rebase-tool/autobuild/prepare [hunk 3]
```diff
@@ -0,0 +1 @@
+export RUSTFLAGS="$RUSTFLAGS -Clink-args=-fuse-ld=bfd"
```
Comment (MingcongBai) [inline app-devel/git-interactive-rebase-tool/autobuild/prepare]: Why was this needed? Please leave a "# FIXME:" comment.

**Review by MingcongBai — CHANGES_REQUESTED** [review]

**Review by MingcongBai — APPROVED** [review]

**Pull request merged**